.nuget/NuGet.Config

```diff
@@ -14,4 +14,7 @@
   <packageSources>
     <add key="nuget.org" value="https://api.nuget.org/v3/index.json" />
   </packageSources>
+  <bindingRedirects>
+    <add key="skip" value="True" />
+  </bindingRedirects> 
```

Comment:
What was happening is that Nuget was creating app.config files for dlls that managed binding redirects. In some cases, this was injecting an unexpected version of the dependency into the scenario (when driving test cases, for example). I am assuming right now that we don't need binding redirects to handle upgrades to later versions of JSON. Is that right?